### ISSUE TITLE
Snapshot files disappear after sync

### ISSUE THREAD

**Issue by tothmartonjeno** [Mimetis/Dotmim.Sync#399]
Server db: SQL Server
Client db: SQLite
I create snapshots in console app:
```
var serverProvider = new SqlSyncChangeTrackingProvider(_connectionString);
            var setup = GetPTSyncSetup();
            var snapshotDirctory = _iconfiguration.GetValue<string>("DotMimSnapshotDirectory");

            var options = new SyncOptions
            {
                SnapshotsDirectory = snapshotDirctory,
                BatchSize = 5000
            };

            var remoteOrchestrator = new RemoteOrchestrator(serverProvider, options, setup);
            var bi = await remoteOrchestrator.CreateSnapshotAsync();
```
In web api startup.cs I setup snapshots:
            
```
            var setup = GetPTSyncSetup();
            var snapshotDirctory = Configuration.GetValue<string>("DotMimSnapshotDirectory");
            var options = new SyncOptions()
            {
                SnapshotsDirectory = snapshotDirctory,
                BatchSize = 5000
            };
            services.AddControllers();
            services.AddMemoryCache();
            var ptConnectionString = Configuration.GetSection("ConnectionStrings")["PackTrack_DV01"];
            services.AddSyncServer<SqlSyncChangeTrackingProvider>(ptConnectionString, setup, options);
```

Xamarin client:

```
public async Task<string> syncronizeWebProxy()
        {
            string syncServerUrl = "http://10.0.2.2:5000/api/sync";
            var serverOrchestrator = new WebClientOrchestrator(syncServerUrl);
            string localDbPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), "pt.db");
            var clientProvider = new SqliteSyncProvider(localDbPath);

            var setup = await GetSyncSetupAsync();
            var clientOptions = new SyncOptions {
                BatchSize = 5000
            };

            var agent = new SyncAgent(clientProvider, serverOrchestrator, clientOptions, setup);
            var progress = new Progress<ProgressArgs>(args =>
                    System.Diagnostics.Debug.WriteLine($"{args.Context.SyncStage}:\t{args.Message}")
                );
            try
            {
                var syncResult = await agent.SynchronizeAsync(SyncType.Reinitialize, progress);
                System.Diagnostics.Debug.WriteLine(syncResult.ToString());
                return syncResult.ToString();
            }
            catch (Exception e)
            {
                return e.Message;
            }
        }
```

After sync the snapshot files disappear. I think this is not the expected operation. Maybe something wrong in my code?

**Comment by Mimetis** [comment]
Please format the code correctly, using Markdown style, it's not readable

**Comment by Mimetis** [comment]
(Thanks for formatting)

Your code seems totally correct to me.
I will make some tests, maybe the `DeleteMetadata` options has a bad behavior somewhere
Thanks for the report !

**Comment by Mimetis** [comment]
Which `DMS` version are you using ?

**Comment by tothmartonjeno** [comment]
0.5.7

**Comment by Mimetis** [comment]
It's resolved in the last release `v0.6`
